src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -27,8 +27,8 @@ interface CaptionProps {
 function Caption({ icon: Icon = AlertTriangle, caption }: CaptionProps) {
   return (
     <>
-      <Icon color="secondary" />
-      <ThemedText.Caption>{caption}</ThemedText.Caption>
+      <Icon color="secondary" size={16.67 / 14} />
```

Comment:
Is this sizing necessary?

Reply:
it is... but it looks like it's not actually being applied

Reply:
was working with the old `largeiconcss`, but that's not here now